deepchecks/vision/utils/image_formatters.py

```diff
@@ -73,3 +79,91 @@ def validate_data(self, batch_data):
             raise DeepchecksValueError('The data inside the iterable must have 1 or 3 channels.')
         if sample.min() < 0 or sample.max() > 255:
             raise DeepchecksValueError('The data inside the iterable must be in the range [0, 255].')
+
+    def sizes(self, batch: List[np.array]):
+        """Return list of tuples of image height and width."""
+        return [self.get_size(img) for img in batch]
+
+    def ratio(self, batch: List[np.array]):
+        """Return list of floats of image height and width ratio."""
+        return [x[0] / x[1] for x in self.sizes(batch)]
+
+    def areas(self, batch: List[np.array]):
+        """Return list of integers of image area (height multiplied by width)."""
+        return [np.prod(self.get_size(img)) for img in batch]
+
+    def brightness(self, batch: List[np.array]):
+        """Calculate brightness on each image in the batch."""
+        if self._is_grayscale(batch) is True:
+            return [img.mean() for img in batch]
+        else:
+            flattened_batch = self._flatten_batch(batch)
+            return [(0.299*img[:, 0] + 0.587*img[:, 1] + 0.114 * img[:, 2]).mean() for img in flattened_batch]
```

Comment:
no existing to_greyscale?

Reply:
Surprisingly not if your not reading from a file or using another class.
Also, as this is done on the flattened version, as I want this to work on sampling, this is better in my opinion.
Thought about extracting to another method, WDYT?